index.js

```diff
@@ -48,17 +72,18 @@ CompassCompilerPlugin.prototype.toTree = function(tree, inputPath, outputPath, i
     var outputFileName = path.basename(outputPaths[file], extension) + extension; // new name for asset
     var outputDir = removeLeadingSlash(path.dirname(outputPaths[file])); // new directory for asset
     var node = new Funnel(compassTree, {
-      srcDir: outputPath,
```

Comment:
So I think the problem is that in regular Ember app projects, the intermediate Compass output file is actually `COMPASS_OUTPUT_TREE/assets/app.css` but for Ember addons, it lives at `COMPASS_OUTPUT_TREE/app.css` for some reason.